hw/dv/tools/dvsim/sim.mk

```diff
@@ -60,31 +60,29 @@ ifneq (${sw_images},)
 	# `sw_images` is a space-separated list of tests to be built into an image.
 	# Optionally, each item in the list can have additional metadata / flags using
 	# the delimiter ':'. The format is as follows:
-	# <path-to-sw-test>:<index>:<flag1>:<flag2>
+	# <Bazel label>:<index>:<flag1>:<flag2>
 	#
 	# If no delimiter is detected, then the full string is considered to be the
-	# <path-to-sw-test>. If 1 delimiter is detected, then it must be <path-to-sw-
-	# test> followed by <index>. The <flag> is considered optional.
+	# <Bazel label>. If 1 delimiter is detected, then it must be <Bazel label>
+	# followed by <index>. The <flag> is considered optional.
 	set -e; \
-	mkdir -p ${sw_build_dir}; \
```

Comment:
@sriyerg since we no longer use the `sw_build_dir` can we remove this from dvsim and all config files?

Reply:
Yes that SGTM - lets remove stuff that we no longer need.

Reply:
cool. I will do so in a follow up PR if thats ok?

Reply:
Actually, I added a commit to this PR with the fix.